src/settings/MappingProfiles/MappingProfilesTransformationsModal/TransformationsField/TransformationsField.js

```diff
@@ -19,16 +22,35 @@ const columnWidths = {
 };
 const visibleColumns = ['isSelected', 'fieldName', 'transformation'];
 
-export const TransformationField = ({ contentData }) => {
+export const TransformationField = ({
+  contentData,
+  isSelectAllChecked,
+  onSelectChange,
+  onSelectAll,
+}) => {
   const intl = useIntl();
 
   const formatter = {
-    isSelected: () => (
-      <Checkbox
-        data-test-select-field
-        aria-label={intl.formatMessage({ id: 'ui-data-export.mappingProfiles.transformations.selectField' })}
+    isSelected: record => (
+      <Field
+        key={record.displayName}
+        name={`transformations[${record.order}].isSelected`}
         type="checkbox"
-      />
+      >
+        { props => (
```

Comment:
Since only one prop is needed, I suggest destructuring here.
```suggestion
        {({ input }) => (
```